style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  font-family: Georgia, serif;
+  font-size: 1.6rem;
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+  margin: 0;
+}
+
+#main {
+  height: 100vh;
+  width: 90vw;
+  right: 10;
+  border: 2p;
+  /* margin: 30px 8px; */
+  padding: 15px;
```

Comment:
Please correct css indentation. 
I would suggest to use proper git workflow and include linters file.

Reply:
Great job pointing out the error for the programmer. I think you can also mention why the changes are required so that the writer can know the changes are needed.

I think you can make use of emojis and gifs to make the code review more human-friendly 👽🤖.